testpmd/hooks/roles/mirror-rh-nfv/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "Mirror index image"
+  shell:
+    cmd: >
+      skopeo copy 
+      --all
+      --authfile {{ mrn_pull_secret }}
+      --dest-tls-verify=false
+      --remove-signatures
+      docker://{{ mrn_index_image }}
+      docker://{{ mrn_mirror_index_image }}
+  register: copy
+  retries: 5
+  delay: 5
+  until:
+    - copy is not failed
+
+- name: "Create temp manifest dir"
+  tempfile:
+    state: directory
+  register: mrn_manifest_dir
+
+- name: "Mirror images from index"
+  shell:
+    cmd: >
+      oc adm catalog mirror
+      -a {{ mrn_pull_secret }}
+      --to-manifests={{ mrn_manifest_dir.path }}
+      {{ mrn_mirror_index_image }}
+      {{ mrn_local_registry }}
+
+- name: "Apply ImageContentSourcePolicy"
```

Comment:
Is this still required after the mirror is added?

Reply:
If the config has been already added it  will not render a new `machineConfig` and the following steps will go really quick.
I think it would be good to leave it just in case someone else is running it without that entry in the ImageContentSourcePolicy already.